src/seals/diagnostics/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""Simple diagnostic environments."""
+
+from seals.diagnostics.risky_path import RiskyPathEnv
```

Comment:
I'd suggest we follow the convention followed in OpenAI Gym which is to:
  - Register all environments in the `__init__.py` file, like in https://github.com/openai/gym/blob/master/gym/envs/__init__.py
  - Do *not* import the submodules defining these environments. This is done lazily when `gym.make` is called.
The advantage of this is (a) submodules can depend on optional dependencies without breaking things if you don't load that environment; (b) it'll keep `import seals` fast.

This isn't a huge deal for these environments since they're all simple Python implementations without external dependencies and which will have minimal overhead, but it's important for other environments we may add in the future which have optional external dependencies like MuJoCo, and I'd rather keep the style consistent throughout.

We should also import `diagnostics` from `src/seals/__init__.py` so that environments are automatically available. This is OK if the `__init__.py` only contains `gym.register` calls which are fast.